File: csharp/examples/KappaNumbers.cs
namespace ClounceMathExamples {
    using ClounceMath;
    using System;

    sealed class KappaNumbers {
        public static void RunSequence() {
            int i = 1;
            int l = 9;
            foreach (double number in Sequences.KappaNumberGenerator(l)) {
                Console.Write("K({0,2}, {1,2}): {2,-30}", l, i, number);
                if (++i % 3 == 0) {
                    Console.WriteLine();
                }
            }
            Console.WriteLine();
            Console.WriteLine();
            Console.WriteLine("Using the direct value calculation");

            for (i = 1; i <= l; ) {
                Console.Write("K({0,2}, {1,2}): {2,-30}", l, i, Sequences.KappaNumber(l, i));
                if (++i % 3 == 1) {
                    Console.WriteLine();
                }
            }
            Console.WriteLine();
        } 
    }
}

File: php/examples/kappa_numbers.php
<?php

require_once('../lib/sequences.php');

$i = 1;
foreach (ClounceMath\Sequences::kappaNumberGenerator(9) as $sequenceNumber) {
    printf("k(%d, %2s): %-30s", 9, $i++, $sequenceNumber);
    if ($i % 3 === 1) {
        echo PHP_EOL;
    }
}

printf(PHP_EOL . "Using direct computation" . PHP_EOL);
for ($i = 1; $i <= 9; ) {
    printf("k(%d, %2s): %-30s", 9, $i, ClounceMath\Sequences::kappaNumber(9, $i));
    if (++$i % 3 === 1) {
        echo PHP_EOL;
    }
}